README.md

```diff
@@ -11,6 +11,45 @@ ImageKit Angular SDK allows you to resize, optimize, deliver, and upload images
 
 ImageKit is complete media storage, optimization, and transformation solution that comes with an image and video CDN. It can be integrated with your existing infrastructure - storage like AWS S3, web servers, your CDN, and custom domain names, allowing you to deliver optimized images in minutes with minimal code changes.
 
+## Breaking changes - Upgrading from 2.x to 3.x version
+3.x version has breaking changes as listed below.
+* Previously, when using this SDK, we had to pass `authenticationEndpoint` which is used by SDK internally for fetching security parameters i.e `signature`, `token`, and `expire`.
```

Comment:
This point is not needed, we have mentioned everything in the next.

Reply:
done removed the first point in [8887b1a](https://github.com/imagekit-developer/imagekit-angular/pull/92/commits/8887b1a64d7edbc8e8fb4fccb37417dce7656d75)

Reply:
done removed the log in [8887b1a](https://github.com/imagekit-developer/imagekit-angular/pull/92/commits/8887b1a64d7edbc8e8fb4fccb37417dce7656d75)